api/src/main/java/org/asynchttpclient/handler/StreamedAsyncHandler.java

```diff
@@ -0,0 +1,19 @@
+package org.asynchttpclient.handler;
```

Comment:
Missing license header. Please grant copyright to "the AsyncHttpClient Project".